src/Fixer/ControlStructure/EmptyLoopBodyFixer.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\ControlStructure;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurableFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolverInterface;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixer\Tokenizer\TokensAnalyzer;
+
+/**
+ * @author SpacePossum
+ */
+final class EmptyLoopBodyFixer extends AbstractFixer implements ConfigurableFixerInterface
+{
+    /**
+     * @internal
+     */
+    public const STYLE_BRACES = 'braces';
+
+    /**
+     * @internal
+     */
+    public const STYLE_SEMICOLON = 'semicolon';
+
+    private $tokenLoopKinds = [T_FOR, T_FOREACH, T_WHILE];
```

Comment:
Can be a constant.

Reply:
that would be at the cost of visibility, unless you mean something else ( https://3v4l.org/mEY0G )

Reply:
I meant
```php
private const TOKEN_LOOP_KINDS = [T_FOR, T_FOREACH, T_WHILE];
```